lib/gems/pending/VMwareWebService/MiqVimInventory.rb

```diff
@@ -64,11 +65,17 @@ def initialize(server, username, password, cacheScope = nil)
         deleteProperty(:HostSystem, "config.dateTimeInfo")
       end
     else
-      if @v4 && cacheScope != :cache_scope_event_monitor
+      if @v4
         @propMap = dupProps(@propMap)
-        addProperty(:VirtualMachine, "runtime.memoryOverhead")
-        deleteProperty(:VirtualMachine, "config.hardware.numCoresPerSocket")
+
+        deleteProperty(:Datastore, "summary.maintenanceMode")
+
+        if cacheScope != :cache_scope_event_monitor
+          addProperty(:VirtualMachine, "runtime.memoryOverhead")
+          deleteProperty(:VirtualMachine, "config.hardware.numCoresPerSocket")
```

Comment:
@agrare 
Hmmm, it's been so long, I can't remember. It could be those `deleteProperty` calls were added at different times, not realizing that a call already existed in another location - but I'm not sure.